Example/CoreDiagnostics/Tests/FIRCoreDiagnosticsTest.m

```diff
@@ -31,10 +33,11 @@
 
 #import "FIRCDLibrary/Protogen/nanopb/firebasecore.nanopb.h"
 
+#import "FIRDiagnosticsDateFileStorage.h"
```

Comment:
Use FIRCDLibrary-relative paths. This makes copybara easier.

Reply:
Updated